dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/cases/TenantAPITest.java

```diff
@@ -79,7 +79,7 @@ public void testDuplicateCreateTenant() {
 
         HttpResponse createTenantHttpResponse = tenantPage.createTenant(sessionId, tenant, 1, "");
 
-        Assertions.assertFalse(createTenantHttpResponse.body().success());
+        Assertions.assertFalse(createTenantHttpResponse.getBody().getSuccess());
```

Comment:
Do you think `getBody()` is neater than `body()` 😉

Reply:
`body()` is neater maybe. It seems the configurations of `lombok` are different in `api test` and `e2e` module from those in other modules. I want them to be consistent.